manager/controllers/app/blueprint_controller.go

```diff
@@ -116,12 +116,12 @@ func (r *BlueprintReconciler) applyChartResource(ctx context.Context, log logr.L
 
 	rel, err := r.Helmer.Status(kubeNamespace, releaseName)
 	if err == nil && rel != nil {
-		rel, err = r.Helmer.Upgrade(chart, kubeNamespace, releaseName, vals)
+		rel, err = r.Helmer.Upgrade(chart, kubeNamespace, releaseName, vals, false)
```

Comment:
Why are we adding Wait flag if we're not using it? Is there a plan to use it? Isn't it too risky to do a synchronic wait as part of reconcile (rather than polling)?

Reply:
Wait will be used in the next PR. 
My understanding is that "wait" does not block the function, but rather means that the release status becomes Deployed only after the resources are ready.